package.json

```diff
@@ -7,7 +7,9 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "./node_modules/.bin/webpack-dev-server --content-base example/ --hot",
-    "prepublish": "NODE_ENV=production ./node_modules/.bin/webpack --config webpack.config.production.js",
+    "prepublish": "npm run minify",
+    "prepare": "npm run minify",
+    "minify": "NODE_ENV=production ./node_modules/.bin/webpack --config webpack.config.production.js",
```

Comment:
`prepublish` will run when publishing to npmjs.com, but since we aren't publishing this on npmjs.com, we need to tell npm to generate the minified build of this package